sass/style.scss

```diff
@@ -1,3 +1,10 @@
+$color-primary:#9356DC;
```

Comment:
does it work as you expected ?

Reply:
+ why is this file called style.scss and not colors or something related to what i can see here and bellow.

Reply:
I'll create a color.scss file and keep the police in style.scss